locations/jclouds/src/test/java/org/apache/brooklyn/location/jclouds/JcloudsLocationTest.java

```diff
@@ -580,8 +578,8 @@ protected String getCreateUserStatementsFor(Map<ConfigKey<?>,?> config) {
                         .put(JcloudsLocationConfig.USER, "bob").put(JcloudsLocationConfig.LOGIN_USER_PASSWORD, "b0b")
                         .putAll(config).build());
 
-        UserCreation creation = jl.createUserStatements(null, jl.config().getBag());
-        return new StatementList(creation.statements).render(OsFamily.UNIX);
+        CreateUserStatements creation = CreateUserStatements.get(jl, null, jl.config().getBag());
```

Comment:
Feels like we could (should?) extract these tests into a `CreateUserStatementsTest` class. But doesn't have to be done in this PR obviously.